doc/source/package-ref.rst

```diff
@@ -135,6 +135,15 @@ Experimental APIs
 .. automodule:: ray.experimental
    :members:
 
+.. _ray-queue-ref:
+
+ray.experimental.queue
+~~~~~~~~~~~~~~~~~~~~~~
+
+.. autoclass:: ray.experimental.queue.Queue
+   :members:
```

Comment:
@ericl is this OK? seems to be useful for those implementing synchronization (see above documentation)

Reply:
Hmm I would actually advocate removing the experimental API section entirely. I think users can just read the source code if they really want to use experimental features.

Reply:
(and we should be aggressive about promoting experimental to mainline).

Reply:
ok removed

Reply:
kk that should be a separate PR then.